src/wasm-lib/kcl/tests/artifact_graph_example_code_offset_planes/ops.snap

```diff
@@ -1,19 +1,44 @@
 ---
 source: kcl/src/simulation_tests.rs
 description: Operations executed artifact_graph_example_code_offset_planes.kcl
-snapshot_kind: text
 ---
 [
   {
     "labeledArgs": {
       "offset": {
+        "value": {
+          "type": "Number",
+          "value": 20.0,
+          "__meta": [
+            {
+              "sourceRange": [
+                35,
+                37,
+                0
+              ]
+            }
+          ]
+        },
         "sourceRange": [
           35,
           37,
           0
         ]
```

Comment:
I think across all of these `value.__meta.sourcRange` and `.sourceRange` are duplicate

Reply:
`value` might not be included, like in the case of a sketch, and it's not trivial to remove a field like meta in the serialized representation without changing it everywhere.

Are you saying you don't need source range in the case of a sketch or other scene object? That would be the only way we could remove it.

Reply:
I removed tag declarators and tag identifiers.

Reply:
Nah `sourceRange` is useful, I was thinking that `value.__meta.sourceRange` was the one that could be removed. Totally fine that can't

Reply:
With `OpKclValue`, we no longer include `__meta`.